addons/account/models/account_reconcile_model.py

```diff
@@ -1011,7 +1011,7 @@ def _check_rule_propositions(self, matched_candidates_values):
             return {'rejected'}
 
         # The statement line will be fully reconciled.
-        residual_balance_after_rec = matched_candidates_values['residual_balance_curr'] + matched_candidates_values['candidates_balance_curr']
+        residual_balance_after_rec = currency.round(matched_candidates_values['residual_balance_curr'] + matched_candidates_values['candidates_balance_curr'])
```

Comment:
indent, for instance:
```suggestion
        residual_balance_after_rec = currency.round(
            matched_candidates_values['residual_balance_curr'] + matched_candidates_values['candidates_balance_curr'])
```

Reply:
shouldn't we round only when we compute `reconciled_percentage_left` (seems safer at first sight) ?